metaseq/modules/feedforward_network.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torch import nn as nn
+from torch.nn import functional as F
+
+from metaseq.modules import Linear, gelu
+from metaseq.modules.fused_bias_gelu import has_fused_bias_gelu, fused_bias_gelu
+
+
+def FeedForwardNetwork(x, fc1, activation_fn, fc2, dropout_module):
```

Comment:
Nit: add a comment here with a high-level description, similar to the encode/decoder classes?